python/whylogs/api/writer/whylabs.py

```diff
@@ -911,6 +935,15 @@ def _get_upload_urls_segmented_reference(self, whylabs_tags, dataset_timestamp:
         res = self._post_log_segmented_reference(request=request, dataset_timestamp=dataset_timestamp)
         return res["id"], res["upload_urls"]
 
+    # TODO: add this to client API
```

Comment:
This is a fake implementation. We need a client entry point that returns a URL for a zipped profile. The integration test for this case will fail until this is implemented, but the unzipped cases are working.